WeakAuras/Prototypes.lua

```diff
@@ -3677,7 +3677,7 @@ Private.event_prototypes = {
         if powerType == 4 then
           table.insert(ret, [[
             local power = GetComboPoints(unit, unit .. '-target')
-            local total = math.max(1, UnitPowerMax(unit, 14))
+            local total = math.max(1, UnitPowerMax(unit, 4))
```

Comment:
it was broken by https://github.com/WeakAuras/WeakAuras2/commit/fa6ae9dbd9c42db35022749c5780cf2d90e7b20b#diff-74981ab15c86143ed223f16f0262d7909123c141f07fa96bd3948b2981668720R3583

Reply:
![image](https://github.com/user-attachments/assets/0863c87a-e449-4404-89da-0ba698a10334)
🤔 what was powertype 14 before it was obsoleted?